pkg/ansible/proxy/proxy_test.go

```diff
@@ -56,7 +56,7 @@ func TestHandler(t *testing.T) {
 		t.Fatalf("Error starting proxy: %v", err)
 	}
 
-	po := createPod("test", "default", mgr.GetConfig())
+	po := createPod("test", "default", mgr.GetConfig(), t)
```

Comment:
Nit: could we actually update these functions to return the error and then do the error check and `t.Fatal` in this function? That would make `createPod` and `deletePod` a bit more re-usable.

While we're at it, I also notice that we pass the `rest.Config` to these functions and create the client each time, which is inefficient. Can we create the client once from the calling function and pass the client into `createPod` and `deletePod` instead?

Reply:
Agreed 100% I will be working in the improvements.